tests/validation/tests/v3_api/scripts/pipeline/Jenkinsfile_ontag_matrix

```diff
@@ -150,6 +155,22 @@ if ( true == via_webhook() && (!(rancher_version ==~ rancher_version_regex)) ) {
                   }
                 }
 
+                // AWS_USER and AWS_AMI are currently not set in the environmental variables.
+                env.CATTLE_TEST_URL = env.CATTLE_TEST_URL.replace('https://', '')
+                env.CONFIG = env.CONFIG.replace('${CATTLE_TEST_URL}', "${env.CATTLE_TEST_URL}")
```

Comment:
we're getting a security warning I think due to these additions. 
```
A secret was passed to "build" using Groovy String interpolation, which is insecure.
 		 Affected argument(s) used the following variable(s): [AWS_ACCESS_KEY_ID, AZURE_SUBSCRIPTION_ID, AZURE_CLIENT_SECRET, AWS_SECRET_ACCESS_KEY]
```

Can you follow the guidelines here to prevent the insecurity? https://www.jenkins.io/doc/book/pipeline/jenkinsfile/#string-interpolation

Reply:
Good catch, looks like it's because I used double quotations and not single quotes, I'll make that change

Reply:
Resolved.